HunterPie/Game/Rise/Data/AbnormalityData.xml

```diff
@@ -299,6 +299,13 @@
                      WithValue="2"
                      Name="ABNORMALITY_BUBBLES_PLUS"
                      Icon="ICON_BUBBLE+"/>
+        <Abnormality Id="ABN_LEECHED"
+                     Offset="8B0"
+                     DependsOn="8B0"
+                     WithValue="-1"
+                     Name="ABNORMALITY_LEECHED"
+                     Icon="ICON_LEECHED"
+                     IsInfinite="True"/>
```

Comment:
Is leeched really infinite or is this just a hacky solution?

Reply:
It basically lasts until a Leeched monster drops. On player side, has no timer.

Reply:
A timer is included in each monster's information.
Can find specific monster using enemy unique id.
A new address needs to be added and the Scan item increases a lot.

Reply:
Nah, it's fine, don't think that extra complexity is worth it.